front_end/html/problem.html

```diff
@@ -85,7 +85,7 @@ <h2 style="display: inline">Expected output:</h2>
             <div id="submissions"></div>
 
             <div>
-              <a href="/edit_problem/{{ course_basics["id"] }}/{{ assignment_basics["id"] }}/{{ problem_basics["id"] }}">Edit problem</a>
+              <a href="/edit_problem/{{ course_basics["id"] }}/{{ assignment_basics["id"] }}/{{ problem_basics["id"] }}">Edit problem</a></br></br></br>
```

Comment:
There's actually no such thing as a closing br tag. But you can have a self-closing tag`<br />`. Perhaps that's what you meant. Please play with it a bit more to see what will work. Please also check on two different browsers to make sure it looks right on both.